hack/vendor.sh

```diff
@@ -85,6 +85,7 @@ clone git github.com/ugorji/go 5abd4e96a45c386928ed2ca2a7ef63e2533e18ec
 clone git github.com/hashicorp/consul v0.5.2
 clone git github.com/boltdb/bolt v1.2.1
 clone git github.com/miekg/dns 75e6e86cc601825c5dbcd4e0c209eab180997cd7
+clone git github.com/docker/docker-credential-helpers 2ccf72627d3d89732442be0de61514c5f30701df https://github.com/calavera/docker-credential-helpers.git
```

Comment:
first, I needs to land in master, it's in my branch waiting for a review :smile: